symfit/core/support.py

```diff
@@ -87,23 +87,27 @@ def f(x, p):
 
     return f
 
-def variables(names):
+def variables(names, **kwargs):
     """
     Convenience function for the creation of multiple variables.
 
-    :param names: string of variable names. Should be comma seperated.
+    :param names: string of variable names.
         Example: x, y = variables('x, y')
+    :param kwargs: kwargs to be passed onto :meth:`sympy.symbols`
+    :return: iterable of :class:`symfit.core.Variable` objects
     """
-    return [Variable(name=name.strip()) for name in names.split(',')]
+    return symbols(names, cls=Variable, seq=True, **kwargs)
 
-def parameters(names):
+def parameters(names, **kwargs):
     """
     Convenience function for the creation of multiple parameters.
 
-    :param names: string of parameter names. Should be comma seperated.
+    :param names: string of parameter names.
         Example: a, b = parameters('a, b')
+    :param kwargs: kwargs to be passed onto :meth:`sympy.symbols`
+    :return: iterable of :class:`symfit.core.Parameter` objects
     """
-    return [Parameter(name=name.strip()) for name in names.split(',')]
+    return symbols(names, cls=Parameter, seq=True, **kwargs)
```

Comment:
You can do some meta-programming thing here by making a symbol_builder which you pass just `Parameter` or `Variable`. It would save you a little repetition, but probably won't be worth the effort.

Reply:
I'm actually thinking of going back to the `sympy` essence; which is to remove `parameters` and `variables` entirely and to update the docs to have users call `symbols(names, cls=Parameter)` directly. It's only slightly more verbose for the user and does provide a lot of clarity.